fields.go

```diff
@@ -26,13 +27,31 @@ func GetStringFieldWithDefault(r *http.Request, fieldName, defaultValue string)
 	return defaultValue
 }
 
+// A multiParamRegex is used to convert Ruby and PHP style array params.
+// PHP uses ["param[0]", "param[1]",..] instead of ["param", "param",..]
+// Ruby uses ["param[]", "param[]",..]
+var multiParamRegex *regexp.Regexp
+
+func init() {
+	multiParamRegex = regexp.MustCompile(`^([a-z:]*)\[[\d]*\]$`)
```

Comment:
Why do you need additional square brackets around `\d`? Just `\d*` should behave the same way.